packages/services/docker-compose.mode.yml

```
@@ -0,0 +1,33 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:latest
```

I think we could use the custom image I created for the github runner here (which extending `postgres:latest` by setting the WAL config like mode needs it)
```suggestion
    image: ghcr.io/latticexyz/postgres-wal-logical:latest
```

---

although, actually it might require additional steps like authorising to the github docker registry. maybe just using the init script would be easier for devs.